packages/itwinui-css/src/input/input.scss

```diff
@@ -3,51 +3,26 @@
 @use '../mixins';
 
 @mixin iui-input {
-  --_iui-input-background-color: var(--iui-color-background);
-  --_iui-input-border-color: var(--iui-color-border-foreground);
-  --_iui-input-text-color: var(--iui-color-text);
+  --_iui-field-color-text-placeholder: var(--iui-color-text-placeholder);
```

Comment:
Since `--_iui-field-color-text-placeholder` does not exist in `iui-field`, this should be renamed or removed (in favor of using `color: var(--iui-color-text-placeholder)` directly).

Reply:
Using `color: var(--iui-color-text-placeholder)` directly.